cinema/models.py

```diff
@@ -35,12 +39,25 @@ def full_name(self):
         return f"{self.first_name} {self.last_name}"
 
 
+def movie_image_path(instance: "Movie", filename: str) -> str:
+    filename = (
+        f"{slugify(instance.title)}-{uuid.uuid4()}"
+        + pathlib.Path(filename).suffix
+    )
+    return str(pathlib.Path("upload/movie/") / filename)
```

Comment:
The `movie_image_path` function is defined here, but the `Movie` model's `image` field uses a static string for `upload_to`. Ensure that this is intentional and consistent with your application's requirements. If you intend to use `movie_image_path`, update the `upload_to` parameter in the `Movie` model accordingly.